cmd/queue/main.go

```diff
@@ -435,9 +435,9 @@ func buildMetricsServer(promStatReporter *queue.PrometheusStatsReporter, protobu
 func metricsHTTPHandler(promStatReporter *queue.PrometheusStatsReporter, protobufStatReporter *queue.ProtobufStatsReporter) http.Handler {
```

Comment:
While you're at it: Should we move this next to the prometheus/protobuf impls. and test it?

Reply:
I actually have a follow-on PR mostly ready to tidy the tests up for the protobuf path (since right now they reach in and directly access the atomic.Value and don't test the HTTP handler at all), think this might make more sense to push down and test as part of that if you're ok with that?

Reply:
(although if we're planning to drop the prometheus scraping stuff in the next release _anyway_ it might be tempting to hold our nose a bit longer..?)

Reply:
We're only dropping the scraping part, not the serving parth.

Reply:
(context from slack: we should dig in and see if observability actually uses this and/or if we can move this to user metrics port once we're only scraping protobuf)